specs/disk_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,111 @@
+= Specification for adding disk details in node hardware inventory
+
+Add disk details also as a part of node hardware inventory
+
+== Problem description
+
+Currently disk details are missing in node hardware inventory. Add disk details
+in node hardware inventory and disk details should contain device_name, fs_type,
+fs_uuid, model, mount_point, parent, size, name, disk_type, used, ssd, vendor,
+disk_id, node_id.
+
+== Use Cases
+
+Tendrl should show the current status of disk.
+
+== Proposed change
+
+* Modify manager module to find disk details and update those details into
+  node_inventory list.
+* Create a new class in persistence module to keep disk details.
+* Populate the object using node_inventory list and persist the object in etcd.
+
```

Comment:
More details should go in here like the command used and output as in the pull request https://github.com/Tendrl/specifications/pull/6